src/Compute/Compute/Manual/PSVirtualMachineScaleSet.cs

```diff
@@ -1,24 +1,3 @@
-// 
-// Copyright (c) Microsoft and contributors.  All rights reserved.
```

Comment:
do we need to keep this stuff? Unsure where it came from.

Reply:
AI generated, noted for ps cmd assistant.

Reply:
I think we should keep the copyright statement

Reply:
Added back in, removed by AI agent. noted for improvements.